### PR TITLE
modify quick_start

### PR DESCRIPTION
**Pull request by bxq2011hust**

### CODE REVIEW

File: docs/tutorials/index.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+##############################################################
```
Comment (wheatli) [inline docs/tutorials/index.rst]: Don't need this ?

File: docs/quick_start.md [hunk 2]
```diff
@@ -1,11 +1,47 @@
...
-- [二进制安装](manual/install.md#二进制安装)
-- [源码编译](manual/install.md#源码编译)
-## [快速体验](tutorials/tutorials.md)
+在开始之前首先需要安装FISCO BCOS运行所需的依赖。FSICO BCOS节点的运行需要一套公私钥以及证书，使用提供的build_chain脚本生成配置时依赖于`openssl`。
+
+一旦安装好提供预编译的二进制文件，该二进制文件可以在Ubuntu 16.04和CentOS 7.2以上版本运行，参考[这里](manual/install.md#二进制安装)安装。同时FISCO BCOS也提供源码编译的[说明文档](manual/install.md#源码编译)，用户可以自由选择安装方式。
+
+为了简化安装与配置，FISCO BCOS提供了[build_chain][build_chain]脚本来帮助用户快读搭建FISCO BCOS联盟链，该脚本默认从GitHub下载最新版本的预编译二进制，也可通过选项指定用户编译的二进制来搭建FISCO BCOS链。
+
+## [关键概念](key_concepts.md)
```
Comment (wheatli) [inline docs/quick_start.md]: do not have to link to key concepts here

File: docs/tutorials/index.rst (hunk 1 above)
Comment (bxq2011hust) [reply docs/tutorials/index.rst]: This is the .crt title, same as `#` in md

File: docs/quick_start.md (hunk 2 above)
Comment (bxq2011hust) [reply docs/quick_start.md]: ok, deleted.

**Pull request merged**